tests/components/homematicip_cloud/test_climate.py

```diff
@@ -633,6 +633,33 @@ async def test_hmip_climate_services(
     # There is no further call on connection.
     assert len(home._connection.mock_calls) == 10
 
+    await hass.services.async_call(
```

Comment:
Could you add the tests to a new test case? It's nicer, if test cases are smaller.

Reply:
I can do that, bit just followed the other tests